runtime/src/snapshot_utils.rs

```diff
@@ -396,6 +402,11 @@ pub fn archive_snapshot_package(
     datapoint_info!(
         "archive-snapshot-package",
         ("slot", snapshot_package.slot(), i64),
+        (
+            "format",
```

Comment:
nit: Can this be renamed to `archive_format` instead?
```suggestion
            "archive_format",
```

Reply:
updated.